compose/src/main/kotlin/io/github/kakaocup/compose/node/action/NodeActions.kt

```diff
@@ -16,7 +17,7 @@ interface NodeActions {
      * Performs a click action on the element represented by the given semantics node.
      */
     fun performClick() {
-        delegate.perform { it.performClick() }
+        delegate.perform(ComposeBaseActionType.PERFORM_CLICK) { performClick() }
```

Comment:
We add additional information about an operation to show this one in logs and give users more data in their Interceptors.